src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/python/PythonInterpreterSelectionDialog.java

```diff
@@ -33,6 +33,7 @@ public PythonInterpreterSelectionDialog(final JsArray<PythonInterpreter> interpr
       setOkButtonCaption("Select");
       
       widgets_ = new WidgetListBox<PythonInterpreterListEntryUi>();
+      widgets_.setHeight("420px");
```

Comment:
Is this related to this PR, or just a bit of UI fit and finish?

Reply:
Related to the PR, in that I saw that after letting more rows enter the UI it became overly tall, and so constraining the height helped it feel more natural here.